go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/Financial-Times/methode-article-image-set-mapper/v2
+
+go 1.13
+
+require (
+	github.com/Financial-Times/go-fthealth v0.0.0-20171204124831-1b007e2b37b7
+	github.com/Financial-Times/go-logger/v2 v2.0.1
+	github.com/Financial-Times/message-queue-go-producer v0.1.1-0.20170622111849-0bb065111416
+	github.com/Financial-Times/message-queue-gonsumer v1.0.0
+	github.com/Financial-Times/service-status-go v0.0.0-20160323111542-3f5199736a3d
+	github.com/Financial-Times/transactionid-utils-go v0.2.0
+	github.com/Financial-Times/uuid-utils-go v0.0.0-20170516110427-e22658edd0f1
+	github.com/gorilla/context v1.1.1 // indirect
+	github.com/gorilla/handlers v0.0.0-20161206055144-3a5767ca75ec
+	github.com/gorilla/mux v1.3.0
+	github.com/hashicorp/go-version v0.0.0-20170202080759-03c5bf6be031 // indirect
+	github.com/jawher/mow.cli v0.0.0-20170430135212-8327d12beb75
+	github.com/satori/go.uuid v1.1.1-0.20170321230731-5bf94b69c6b6
+	github.com/sirupsen/logrus v1.4.2
```

Comment:
Why update the version of `logrus`?

Reply:
Because we get an error otherwise.

Reply:
What kind of error? Is it related to the kafka library or to the migration to modules?

Reply:
The old version of logrus can't work with the new kafka library.